jmix-reports/reports-flowui/src/main/java/io/jmix/reportsflowui/view/run/ReportTableView.java

```diff
@@ -105,13 +104,27 @@ public class ReportTableView extends StandardView {
     @Autowired
     protected Notifications notifications;
 
-
-    protected Report report;
     protected String templateCode;
     protected Map<String, Object> reportParameters;
     protected InputParametersFragment inputParametersFrame;
+    protected ReportOutputDocument document;
+
+    /**
+     * @deprecated use {@link #document}
+     */
+    @Deprecated(since = "2.1.3", forRemoval = true)
+    protected Report report;
+
+    /**
+     * @deprecated use {@link #document}
+     */
+    @Deprecated(since = "2.1.3", forRemoval = true)
```

Comment:
If we include the issue to Jmix 2.2 then the `since` attribute should be 2.2.0, not 2.1.3